.pipelines/release.yml

```diff
@@ -20,429 +23,423 @@ parameters:
+            solution: 'src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.csproj'
+            vsVersion: 17.0
+            # The arguments should be the same as the ones for Settings; make sure they are.
+            msbuildArgs: >-
+              /target:Publish
+              /p:Configuration=$(BuildConfiguration);Platform=$(BuildPlatform);AppxBundle=Never
+              /p:VCRTForwarders-IncludeDebugCRT=false
+              /p:PowerToysRoot=$(Build.SourcesDirectory)
+              /p:PublishProfile=InstallationPublishProfile.pubxml
+            platform: $(BuildPlatform)
+            configuration: $(BuildConfiguration)
+            maximumCpuCount: true
+
+        - task: VSBuild@1
+          displayName: Publish File Locksmith UI for Packaging
+          inputs:
+            solution: 'src/modules/FileLocksmith/FileLocksmithUI/FileLocksmithUI.csproj'
+            vsVersion: 17.0
+            # The arguments should be the same as the ones for Settings; make sure they are.
+            msbuildArgs: >-
+              /target:Publish
+              /p:Configuration=$(BuildConfiguration);Platform=$(BuildPlatform);AppxBundle=Never
+              /p:VCRTForwarders-IncludeDebugCRT=false
+              /p:PowerToysRoot=$(Build.SourcesDirectory)
+              /p:PublishProfile=InstallationPublishProfile.pubxml
+            platform: $(BuildPlatform)
+            configuration: $(BuildConfiguration)
+            maximumCpuCount: true
+
+        # Check if deps.json files don't reference different dll versions.
+        - task: PowerShell@2
+          displayName: Audit deps.json files for all applications
+          inputs:
+            filePath: '.pipelines/verifyDepsJsonLibraryVersions.ps1'
+            arguments: -targetDir '$(build.sourcesdirectory)\$(BuildPlatform)\$(BuildConfiguration)'
+            pwsh: true
+
+        # Check if asset files on the main application paths are playing nice and avoiding basic conflicts.
+        - task: PowerShell@2
+          displayName: Audit base applications path asset conflicts
+          inputs:
+            filePath: '.pipelines/verifyPossibleAssetConflicts.ps1'
+            arguments: -targetDir '$(build.sourcesdirectory)\$(BuildPlatform)\$(BuildConfiguration)'
+            pwsh: true
+
+        - task: PowerShell@2
+          displayName: Audit WinAppSDK applications path asset conflicts
+          inputs:
+            filePath: '.pipelines/verifyPossibleAssetConflicts.ps1'
+            arguments: -targetDir '$(build.sourcesdirectory)\$(BuildPlatform)\$(BuildConfiguration)\WinUI3Apps'
+            pwsh: true
+
+      #### MAIN SIGNING AREA
+      # reference https://dev.azure.com/microsoft/Dart/_git/AppDriver?path=/ESRPSigning.json&version=GBarm64-netcore&_a=contents for winappdriver
+      # https://dev.azure.com/microsoft/Dart/_git/AppDriver?path=/CIPolicy.xml&version=GBarm64-netcore&_a=contents
+
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
```

Comment:
Just keeping us up to date with the newest ESRP version; this one requires NET 6 instead of NET 2.1 (whoa)